docs/tutorials/podman_tutorial.md

```diff
@@ -80,7 +80,7 @@ $ sudo podman top <container_id>
 ### Checkpointing the container
 Checkpointing a container stops the container while writing the state of all processes in the container to disk.
 With this a container can later be restored and continue running at exactly the same point in time as the
-checkpoint. This capability requires CRIU 3.11 or later installed on the system.
+checkpoint. This capability requires CRIU 3.11 or later installed on the system. Not supported on rootless.
```

Comment:
"This feature is not supported as rootless; as such, if you wish to try it, you'll need to re-create your container as root, using the same command but with `sudo`.